privaterelay/views.py

```diff
@@ -224,7 +225,9 @@ def _token_updater(new_token):
         'client_secret': client_secret,
     }
 
-    expires_in = (social_token.expires_at - timezone.now()).total_seconds()
+    expires_in = (
+        social_token.expires_at - datetime.now(timezone.utc)
```

Comment:
Just wanted to make sure that the `social_token.expires_at` is also set to UTC timezone?